src/sql/workbench/services/objectExplorer/browser/treeUpdateUtils.ts

```diff
@@ -405,7 +405,7 @@ export class TreeUpdateUtils {
 		// Map the indices of profiles that share the same connection name.
 		for (let i = 0; i < inputList.length; i++) {
 			// do not add if the profile is still loading as that will result in erroneous entries.
-			if (inputList[i].hasServerCapabilities && inputList[i].hasLoaded()) {
+			if (inputList[i].serverCapabilities && inputList[i].hasLoaded()) {
```

Comment:
@smartguest Also noticed that this was incorrect too - you're just checking if the function exists on the object. Not actually calling it.